services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> str:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]